apps/juxtaposition-ui/src/api/post.ts

```diff
@@ -63,3 +64,21 @@ export async function getPostById(tokens: UserTokens, post_id: string): Promise<
 	const post = await apiFetchUser<PostDto>(tokens, `/api/v1/posts/${post_id}`);
 	return post;
 }
+
+export async function getPostsByPoster(tokens: UserTokens, poster_pid: number, offset: number): Promise<PageDto<PostDto> | null> {
+	const params = new URLSearchParams({
+		posted_by: poster_pid.toString(),
+		offset: offset.toString()
+	});
+	const posts = await apiFetchUser<PageDto<PostDto>>(tokens, `/api/v1/posts?${params}`);
```

Comment:
This now also gets replies and not just top posts (previous query had a `parent: null` check).

We probably need another query param `has_parent: boolean | undefined`